TestSwitchApi/TestSwitchApi/DataMigrations/005_AddAdminUserAndSessionIDTables.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE TABLE public."AdminUsers" (
+                                     "Id" BIGSERIAL NOT NULL PRIMARY KEY UNIQUE,
+                                     "Email" VARCHAR(256) NOT NULL UNIQUE,
+                                     "HashedPassword" VARCHAR(128) NOT NULL,
+                                     "PasswordSalt" VARCHAR(24) NOT NULL
+);
+
+CREATE TABLE public."AdminUserSessions"(
+                                        "Id" VARCHAR(128) NOT NULL PRIMARY KEY UNIQUE,
+                                        "AdminUserID" BIGINT NOT NULL,
+                                        "SessionStart" VARCHAR(128) NOT NULL,
+                                        "SessionEnd" VARCHAR(128) NOT NULL
```

Comment:
These should _probably_ be either timestamps or dates - https://www.postgresql.org/docs/9.1/datatype-datetime.html